### ISSUE TITLE
Documentation to show definition statement for typing new types

### ISSUE THREAD

**Issue by m2march** [davidhalter/jedi#1205]
Hi. Thanks so much for developing Jedi. It's wonderful to have so much power on python. 

I wanted to get a new feature from Jedi and I was hoping to get some pointers on how I might manage to do that. 

The new feature would be to add to the documentation information of type redefinitions as it is usually done with typing in python 3.

For example:

```
from typing import NewType, Dict, List, Tuple
MyType = NewType('MyType', Dict[int, str])
MyType2 = List[Tuple[int, str]]

def foo(a: MyType, b: MyType2):
    pass
```

I would like to find the assigment statement for `MyType` or `MyType2`  and add that as part of the definition of these classes so I can see the mapping in the documentation.

Thanks in advance,
March

**Comment by davidhalter** [comment]
I'm working on #1096 currently, so this is pretty close and maybe I'll do this as well.

**Comment by m2march** [comment]
Alright. Let me know if I can help with anything. I'll let you know if I come up with sth.

**Comment by davidhalter** [comment]
Ok. The `typeshed` branch is in a good state, now. The tests are passing.

It would be great if you contributed this feature. It's probably not that hard. You can just checkout the `typeshed` branch, open `evaluate/context/typing.py` and add a new `elif` in `_remap`. Good luck!

**Comment by m2march** [comment]
Thanks for getting in touch. I'll try to look into it

**Comment by davidhalter** [comment]
This has been implemented in #1344.
